### PR TITLE
[ts-sdk] Fix for Node.js 20

### PR DESCRIPTION
**Pull request by wkozyra95**
https://github.com/node-fetch/node-fetch/issues/1735

### CODE REVIEW

File: ts/live-compositor/src/components/Tiles.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ function sceneBuilder(props: TilesProps, children: SceneComponent[]): Api.Compon
     children: children.map(sceneComponentIntoApi),
     width: props.width,
     height: props.height,
-    background_color_rgba: props.backgroundColor,
+    background_color_rgba: props.backgroundColor && intoApiRgbaColor(props.backgroundColor),
```
Comment (wkozyra95) [inline ts/live-compositor/src/components/Tiles.ts:66]: unrelated to the node issue (it does not seem to be worth to create separate PR)

**Review by noituri — APPROVED** [review]

**Review by WojciechBarczynski — APPROVED** [review]
